src/stacks-hierarchy/Root_ScanQrCodeScreen.tsx

```diff
@@ -0,0 +1,126 @@
+import {StyleSheet} from '@atb/theme';
+import {MapTexts, useTranslation} from '@atb/translations';
+import React, {useCallback, useEffect, useState} from 'react';
+import {Camera} from '@atb/components/camera';
+
+import {RootStackScreenProps} from '@atb/stacks-hierarchy/navigation-types';
+import {ScreenContainer} from './Root_ParkingViolationsReporting/components/ScreenContainer';
+import {ParkingViolationTexts} from '@atb/translations/screens/ParkingViolations';
+import {useGetIdsFromQrCodeMutation} from '@atb/mobility/queries/use-get-ids-from-qr-code-mutation';
+import {useIsFocusedAndActive} from '@atb/utils/use-is-focused-and-active';
+import {Alert} from 'react-native';
+
+import {useGeolocationState} from '@atb/GeolocationContext';
+import {AutoSelectableBottomSheetType, useMapState} from '@atb/MapContext';
+import {IdsFromQrCodeResponse} from '@atb/api/types/mobility';
+
+export type Props = RootStackScreenProps<'Root_ScanQrCodeScreen'>;
+
+export const Root_ScanQrCodeScreen: React.FC<Props> = ({navigation}) => {
+  const styles = useStyles();
+  const {t} = useTranslation();
+
+  const isFocused = useIsFocusedAndActive();
+  const {currentCoordinatesRef} = useGeolocationState();
+  const {setBottomSheetToAutoSelect} = useMapState();
+  const [hasCapturedQr, setHasCapturedQr] = useState(false);
+
+  const {
+    mutateAsync: getIdsFromQrCode,
+    isLoading: getIdsFromQrCodeIsLoading,
+    isError: getIdsFromQrCodeIsError,
+  } = useGetIdsFromQrCodeMutation();
+
+  const alertResultError = useCallback(() => {
+    Alert.alert(
+      t(MapTexts.qr.notFound.title),
+      t(MapTexts.qr.notFound.description),
+      [
+        {
+          text: t(MapTexts.qr.notFound.ok),
+          style: 'default',
+          onPress: navigation.goBack,
+        },
+      ],
+    );
+  }, [navigation, t]);
```

Comment:
There is something a little iffy here. Now if the language changes, the `alertResultError` will change, which might trigger the Alert on unexpected times. It is a corner case, but the thought experiment exposes that something should be looked into.

Reply:
Hmm, it's only triggered by the `onCapture` event, not used e.g. in a `useEffect`. So I don't think it's a problem?

Reply:
Hopefully the final changes have been made to this PR now - with the updates from master merged into it and updated usage of global variables for both `t` and `coordinates` in this component.
Any further comments before merge?

Reply:
Go ahead!